dashboard/package.json

```diff
@@ -21,6 +21,7 @@
     "@logdna/logger": "^2.6.6",
     "@next/font": "13.1.6",
     "@sentry/react": "^7.37.1",
+    "@types/react-icons": "^3.0.0",
```

Comment:
hmm, aren't the hero icons enough for this?

Reply:
I checked hero icons and it does not contain the one we need here.